src/components/ComposePage/GroupsPage/ModalBox/groupsModalBox.tsx

```diff
@@ -131,28 +143,40 @@ console.log("lastIndiceslastIndices",lastIndices)
 
   function contentToggle() {
     setDisplayAttributeSection(true);
-    setDisplayAttributeSection(true)
-    setRender(!render)
-    setAttrName("Attribute Name")
+    setDisplayAttributeSection(true);
+    setRender(!render);
+    setAttrName("Attribute Name");
     const newLastIndices = [...lastIndices];
-    newLastIndices.push({name: "Attribute Name", type: "VARCHAR"})
+    newLastIndices.push({ name: "Attribute Name", type: "VARCHAR" });
     setLastIndices(newLastIndices);
   }
 
-  function getValues(attributeName:string,selectedDataType:string){
-        setAttrName(attributeName)
-        setDatatype(selectedDataType)
-        selectGroupdataData.slice(-1)[0] = {name: attributeName, type: selectedDataType}
-        lastIndices[lastIndices.length-1] = {name: attributeName, type: selectedDataType}
-       }
+  function getValues(attributeName: string, selectedDataType: string) {
+    setAttrName(attributeName);
+    setDatatype(selectedDataType);
+    selectGroupdataData.slice(-1)[0] = {
+      name: attributeName,
+      type: selectedDataType,
+    };
+    lastIndices[lastIndices.length - 1] = {
+      name: attributeName,
+      type: selectedDataType,
+    };
+  }
 
-  const handleRowClick = (node:any) => {
-    console.log("Getting innnnnn");
+  const handleRowClick = (node: any) => {
     setSelectedNodeDetails([node]);
-    handleAddAttributeDetails(lastIndices);
-    console.log("Data getting printed on clicking the attribute", lastIndices);
-    console.log("selectedNodeDetails", selectedNodeDetails);
-    console.log("Getting outttttttt");
+
```

Comment:
use condition operator (?) and remove commented lines

Reply:
Done